db/migrate/20210707135850_create_models.rb

```diff
@@ -2,8 +2,9 @@ class CreateModels < ActiveRecord::Migration[6.1]
   def change
     create_table :models do |t|
       t.string :name, null: false
+      t.references :brand, null: false, foreign_key: true
 
       t.timestamps
     end
   end
-end
+end
```

Comment:
Should add a new line at the end of the file. Rule not in the paper.
![image](https://user-images.githubusercontent.com/48946963/124847384-f318b980-dfc4-11eb-9ce8-c0a6a98b3501.png)

Reply:
Done!